tests/event_bus_spy.py

```diff
@@ -169,7 +209,7 @@ def assert_events_occured(self, events, in_order=True):
 
     def assert_events_exactly_occured(self, events):
         events = self._cook_events_params(events)
-        assert self.events == events
+        assert events == self.events
```

Comment:
Is the change related to the new `__eq__` ? If so a comment should be added

Reply:
Yes, it's related to the change made in https://github.com/Scille/parsec-cloud/pull/1950/files#diff-65728c3d9bd60f81308060373b1016f2f7cadc9583db97f29f5ebe3b6b7db1c1R60